tests/integration/allocator_integration_tests.cpp

```diff
@@ -130,6 +130,28 @@ TEST_P(AllocatorTest, get_allocator_records)
   m_allocator->deallocate(data);
 }
 
+TEST_P(AllocatorTest, getCurrentSize)
```

Comment:
Does this capture all allocators and strategies?

Reply:
It captures all basic allocators. We don't have tests for all strategies yet, but the DynamicPool is already tested.